frontend/internal-packages/schema-bench/benchmark-workspace-default/execution/input/case-001.json

```diff
@@ -1,3 +1,3 @@
 and total order amount.\n2. Items are received from suppliers based on the purchase order.\n\n   * Each receipt is assigned a receipt number and the receipt date is recorded.\n   * Each item received is assigned a receipt detail number. A purchase order detail corresponds to a single receipt (no split deliveries).\n   * A label with the receipt number and detail number is attached to the item packaging and handed over to the production team.\n\n---\n\n**[Conceptual Data Model and Relational Schema for Current Operations]**\n\nThe conceptual data model is shown in Figure 1.\nThe relational schema is as follows, though the seven relations on the right have incomplete PK/FK definitions: employee_affiliation, model, model_item, prototype_project_item, request_for_quotation, quotation_detail, and purchase_order_detail.\n\n---\n\nFigure 1.\n\n\nemployee_affiliation(\n  employee_code,\n  employee_name,\n  affiliated_organization_code,\n  affiliated_organization_name,\n  parent_organization_code,\n  parent_organization_name,\n  position_code,\n  position_name,\n  reporting_employee_code,\n  reporting_employee_name\n)\n\nbusiness_partner(\n  pk(business_partner_code),\n  business_partner_name,\n  business_partner_category,\n  address\n)\n\nclient(\n  pk(business_partner_code),\n  order_amount_last_year\n)\n\nsupplier(\n  pk(business_partner_code),\n  procurement_amount_last_year\n)\n\nbrand(\n  pk(brand_code),\n  brand_name\n)\n\nitem_category(\n  pk(item_category_code),\n  item_category_name\n)\n\nitem(\n  pk(manufacturer_part_number),\n  brand_code fk,\n  item_category_code fk\n)\n\nhandled_brand(\n  pk(business_partner_code, brand_code)\n)\n\nprototype_project(\n  pk(prototype_project_number),\n  prototype_project_name,\n  business_partner_code fk,\n  product_purpose,\n  prototype_project_registration_date\n)\n\nmodel(\n  pk(model_name),\n  production_quantity,\n  client_requested_delivery_date,\n  design_drawing_number\n)\n\nmodel_item(\n  pk(model_name, manufacturer_part_number),\n  required_quantity_per_unit\n)\n\nprototype_project_item(\n  pk(prototype_project_number, manufacturer_part_number),\n  total_required_quantity\n)\n\nrequest_for_quotation(\n  pk(rfq_number),\n  rfq_date\n)\n\nrfq_detail(\n  pk(rfq_number, rfq_detail_number),\n  manufacturer_part_number fk,\n  required_procurement_quantity,\n  desired_delivery_date\n)\n\nquotation(\n  pk(rfq_number),\n  quotation_number,\n  quotation_validity_period,\n  quotation_date\n)\n\nquotation_detail(\n  pk(quotation_detail_number),\n  rfq_detail_number fk,\n  unit_price,\n  deliverable_date\n)\n\npurchase_order(\n  pk(purchase_order_number),\n  purchase_order_date,\n  total_purchase_order_amount\n)\n\npurchase_order_detail(\n  pk(purchase_order_number, purchase_order_detail_number),\n  specified_delivery_date\n)\n\nreceipt(\n  pk(receipt_number),\n  receipt_date\n)\n\nreceipt_detail(\n  pk(receipt_number, receipt_detail_number),\n  purchase_order_number fk,\n  purchase_order_detail_number fk\n)\n"
```

Comment:
It’s not perfect, but I don’t think this will make things worse, so let me give it a try.










- before: Perform conceptual data modeling...
- after: design the schema: ...